### ISSUE TITLE
Feature Request: Write log file

### ISSUE THREAD

**Issue by Sebbo94BY** [TS3Tools/TS3Monitor#2]
The script should also write a log file, if required.

The admin will be able to turn it on or off in the config.

**Comment by Sebbo94BY** [comment]
The script will now also write a log file to '/var/log/ts3tools/ts3monitor.log' by default:
```
$ cat /var/log/ts3tools/ts3monitor.log
Mai  4 16:34:01 ipdc-vs00019 CRON[39371]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor ts3server --path /home/teamspeak) RESULT (Server is running)
Mai  4 16:38:02 ipdc-vs00019 CRON[39970]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor tsdnsserver --path /home/teamspeak/tsdns) RESULT (CRASHED - Restarted service)
Mai  4 16:38:02 ipdc-vs00019 CRON[39966]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor ts3server --path /home/teamspeak) RESULT (Server is running)
Mai  4 16:39:01 ipdc-vs00019 USER[40538]: (sebastian) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor ts3server --path /home/teamspeak-ipv6) RESULT (Server is running)
Mai  4 16:39:01 ipdc-vs00019 CRON[40713]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor tsdnsserver --path /home/teamspeak/tsdns) RESULT (OK - service is running)
Mai  4 16:39:02 ipdc-vs00019 CRON[40687]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor ts3server --path /home/teamspeak) RESULT (Server is running)
Mai  4 16:40:01 ipdc-vs00019 CRON[41358]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor tsdnsserver --path /home/teamspeak/tsdns) RESULT (OK - service is running)
Mai  4 16:40:01 ipdc-vs00019 CRON[41355]: (cron) CMD (/home/sebastian/github/TS3Monitor/TS3Monitor ts3server --path /home/teamspeak) RESULT (Server is running)
```
The log file format is similar to the `syslog`. This allows you a better parsing for a Syslog server and automation for example.

In the example above, you can see these fields:

- Timestamp
- Short Hostname
- Type of execution: CRON(job) or by an USER
- Process ID
- Real user, who has executed the script (if you log in via SSH and change user, it will still display the first logged in username via SSH)
- CoMmanD with all used parameters in brackets
- Result in brackets

If you want to disable this logging, you can change/add this variable in/to your 'configs/config.all' file:
`WRITE_TS3MONITOR_LOG=false`

**Comment by Sebbo94BY** [comment]
This FR will be released with version 1.1.0.
